src/components/Heading/template.pug

```diff
@@ -1,21 +1,21 @@
 mixin content
     +if('level === 1')
-        h1(class="heading {hoverableClass} {displayClass} {asLink ? '' : className}")
+        h1(use:forwardAction bind:this="{forwardRef}" class="heading {hoverableClass} {displayClass} {asLink ? '' : className}")
```

Comment:
@georgeciubotaru @markholdex what's the use case when we are using Headings with tooltip 🤔 ?

Reply:
@zolotokrylin we don't have now any use-case. That's for future use